draft-ietf-quic-transport.md

```diff
@@ -2678,12 +2678,9 @@ Packet number encoding at a sender and decoding at a receiver are described in
 
 ## Frames and Frame Types {#frames}
 
-The payload of QUIC packets, after removing packet protection, commonly consists
-of a sequence of frames, as shown in {{packet-frames}}.  Version Negotiation,
-Stateless Reset, and Retry packets do not contain frames.
-
-<!-- TODO: Editorial work needed in this section. Not all packets contain
-frames. -->
+The payload of QUIC packets, after removing packet protection, commonly consist
```

Comment:
```suggestion
The payload of a QUIC packet, after removing packet protection, commonly consists
```

or "payload of QUIC packets, ... consists"

Reply:
... maybe remove "commonly" as well, since the exceptions are listed next anyway?